src/lib.rs

```diff
@@ -38,6 +38,34 @@ macro_rules! either_mut {
     )
 }
 
+/// Helper macro for unwrapping the left side of an `Either` while returning
```

Comment:
The description is a bit weird, knowing that left and right cases are exclusive of each other, so saying "if there is no left one" is weird. Maybe it can be reworded a bit?